src/main/java/com/airlines/dao/impl/CrewDaoImpl.java

```diff
@@ -0,0 +1,79 @@
+package com.airlines.dao.impl;
+
+import com.airlines.dao.CrewDao;
+import com.airlines.dao.DbConnector;
+import com.airlines.exception.DaoOperationException;
+import com.airlines.model.Citizenship;
+import com.airlines.model.CrewMember;
+import com.airlines.model.Position;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CrewDaoImpl implements CrewDao {
+
+    private final static String SELECT_CREW_MEMBERS_BY_CREW_ID_QUERY = "SELECT * FROM crew_member WHERE crew_id = ?;";
+    private final static String SELECT_CREW_MEMBERS_BY_CREW_NAME_QUERY = """
```

Comment:
what point for """ ?

Reply:
This is the auto code for the text block.